_templates/navbar.html

```diff
@@ -1,7 +1,6 @@
-<a href="{{pathto('install')}}">Install</a> · 
-<a href="{{pathto('documentation')}}">Docs</a> ·
-<a href="{{pathto('videos')}}">Videos</a> · 
+<a href="http://jupyter.readthedocs.org/en/latest/install.html" target="_blank">Install</a> · 
```

Comment:
Seem weird, IPython should link to IPython install doc, not jupyter. the IPython install doc does have a warning that link to Jupyter if you are looking for the notebook.